test/unit/list/index.test.tsx

```diff
@@ -402,3 +402,14 @@ test('renders a list with a nav tag', () => {
   const wrapper = shallow(<List tag='nav'>{children()}</List>);
   assert.equal(wrapper.type(), 'nav');
 });
+
+test('renders a list with children which is not DOM', () => {
+  const wrapper = shallow(<List>
+    {}
+    {false}
+    {'text'}
+    {null}
+    {undefined}
+  </List>);
+  assert.isNotEmpty(wrapper);
```

Comment:
I think it it more accurate to find how many listitem nodes would be contained in the wrapper. I think this the length should be 1, correct?

Reply:
Yeah you are right!
I added code that checks count of child nodes.